packages/cli/src/commands/rules/execute.ts

```diff
@@ -0,0 +1,36 @@
+import { flags } from '@oclif/command'
+
+import { APICommand } from '@smartthings/cli-lib'
+
+import { chooseRule, getRuleWithLocation } from '../../lib/commands/rules/rules-util'
+
+
+export default class RulesExecuteCommand extends APICommand {
+	static description = 'execute a rule\'s actions'
```

Comment:
We could leave "actions" out as a detail and just say `execute a rule` or `Trigger Rule execution` as the API spec says.

Reply:
Fixed this and I also added examples.